src/sentry/web/urls.py

```diff
@@ -437,13 +437,6 @@ def init_all_applications():
         react_page_view,
         name='sentry-organization-stats'
     ),
-    url(
-        r'^organizations/(?P<organization_slug>[\w_-]+)/teams/new/$', react_page_view),
-    url(
-        r'^organizations/(?P<organization_slug>[\w_-]+)/teams/(?P<team_slug>[\w_-]+)/remove/$',
-        RedirectView.as_view(url='/settings/%(organization_slug)s/teams/%(team_slug)s/settings'),
-        name='sentry-remove-team-old'
-    ),
```

Comment:
This was recently added to keep any bookmarks working. It is probably safe to remove though.

Reply:
ah might be my mistake on the rebase -- but yeah I think we're fine not auto redirecting this (or rather, the frontend should be able to do it itself)